rootchpasswd/Dockerfile.rootchpasswd

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi:latest
+
+RUN echo 'root:Asdf_123' | chpasswd && \
```

Comment:
This is interesting. Would you expect a secret tool to detect Asdf_1234 here? That'd be something I haven't seen before.